addons/sale_project/models/sale_order.py

```diff
@@ -193,7 +193,10 @@ def _timesheet_create_project(self):
 
         # Avoid new tasks to go to 'Undefined Stage'
         if not project.type_ids:
-            project.type_ids = self.env['project.task.type'].create({'name': _('New')})
+            stage_new = self.env['project.task.type'].search([('name', '=', _('New'))], limit=1)
```

Comment:
Searching on the translated name won't work. We should search if there is a first stage, that's all.